client/components/common/jumbotron.js

```diff
@@ -13,10 +9,11 @@ import socialLogo from '../../assets/img/vector_blue_flower_logo.png'
 import { getAllClubs } from '../../actions/socialClubActions';
 
 class Jumbotron extends Component {
+
   componentDidMount() {
     this.props.getAllClubs();
-    window.scrollTo(0, 0);
   }
+
   render() {
     const defaultImageUrl = 'https://www.omnihotels.com/-/media/images/hotels/pueave/hotel/pueave-omni-puerto-aventuras-beach-resort.jpg?h=660&la=en&w=1170';
```

Comment:
I'm not sure we're still using this but if we aren't, let's remove it.

Reply:
Ok.. noted